src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Packages.props

```diff
@@ -1,47 +1,43 @@
-    <!--#endif-->
+    <PackageVersion Include="coverlet.collector" Version="3.2.0" />
+    <PackageVersion Include="FluentAssertions" Version="6.8.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="$msftextlibversion$" />
     <PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="4.2.0" />
-    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.3.0" />
-    <!--#if (msftextv6) -->
-    <PackageVersion Include="Microsoft.Windows.Compatibility" Version="6.0.0" />
-    <!--#endif-->
-    <!--#if (msftextv7) -->
-    <PackageVersion Include="Microsoft.Windows.Compatibility" Version="7.0.0" />
-    <!--#endif-->
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.4.0" />
+    <PackageVersion Include="Microsoft.Windows.Compatibility" Version="$msftextlibversion$" />
     <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
     <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.2.221209.1" />
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageVersion Include="NUnit" Version="3.13.2" />
-    <PackageVersion Include="NUnit3TestAdapter" Version="4.0.0" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageVersion Include="NUnit" Version="3.13.3" />
+    <PackageVersion Include="NUnit3TestAdapter" Version="4.3.1" />
+    <!--#if (server && use-serilog)-->
+    <PackageVersion Include="Serilog.AspNetCore" Version="6.1.0" />
+    <!--#endif-->
     <PackageVersion Include="SkiaSharp.Skottie" Version="2.88.3" />
     <PackageVersion Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.3" />
+    <!--#if (server)-->
+    <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+    <!--#endif-->
     <PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
     <PackageVersion Include="Uno.Extensions.Authentication" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Authentication.WinUI" Version="255.255.255.255" />
-    <!--#if (configuration)-->
+    <!--#if (use-configuration)-->
     <PackageVersion Include="Uno.Extensions.Configuration" Version="255.255.255.255" />
     <!--#endif-->
     <PackageVersion Include="Uno.Extensions.Core" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Hosting" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Hosting.WinUI" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Http" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Http.Refit" Version="255.255.255.255" />
-    <!--#if (localization)-->
+    <!--#if (use-localization)-->
     <PackageVersion Include="Uno.Extensions.Localization" Version="255.255.255.255" />
     <PackageVersion Include="Uno.Extensions.Localization.WinUI" Version="255.255.255.255" />
     <!--#endif-->
-    <!--#if (uselogging)-->
+    <!--#if (use-logging)-->
```

Comment:
Missing "endif"

Reply:
no the endif for this is on line 45. It wraps the `if serilog`